### DISCUSSION TITLE
borgbackup-fuse: An unsatisfied requirement failed this build

### DISCUSSION THREAD

**Discussion by pin00b** [borgbase/vorta#906] [General]
During my regular brew update/upgrade, I got the given error message:

<img width="568" alt="Bildschirmfoto 2021-03-01 um 09 19 12" src="https://user-images.githubusercontent.com/61549899/109471509-25436900-7a71-11eb-9645-69260947b402.png">

I didn't follow the advice to install macfuse first, since there were some difficulties in the past on using macfuse together with vorta/borgbackup... – Any hints & advices to solve the problem very much appreciated!  Thx

**Comment by m3nu** [comment]
Yeah, you will need the macFUSE formula installed first.

If you don't need FUSE, it's best to just use the normal borgbackup formula – `$ brew install borgbackup`

**Reply by pin00b** [reply]
Ah, so right now it's without complications to install macFUSE and also necessary for borgbackup-fuse? I thought this was the reason for conflicting results in the past? I'm confused, sorry.

**Reply by m3nu** [reply]
Which results? macFUSE is the only option to get FUSE on macOS at all. Especially for M1 machines.

**Reply by pin00b** [reply]
OK, sorry, dangerous self educated half-knoledge, not a dev. Thought of the incompatibilities of macFUSE and Vorta in the closer past, need for tap-usage etc., but it works right now, mounting not yet tested. Thank you for answering as fast as hell 😉

**Reply by m3nu** [reply]
The Borg binary shipped with Vorta may not work with macFUSE. Didnt try yet. You will need macFUSE either way.

Personally, I’d just install all three via Homebrew and also get updates. Just as described in the docs:

https://vorta.borgbase.com/install/macos/
